Animal.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FirstCourseApp
+{
+    class Animal
+    {
+        public string name = "";
```

Comment:
1. Breaks encapsulation OOP principle
2. Name of the public methods, properties.. should start with Capital case